doc/source/docs/user_guide/interactive_mapping.ipynb

```diff
@@ -143,7 +146,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.2"
+   "version": "3.11.0"
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "b1fe2ae8565152c84d3dbd08488d3746f754c9bdf2de9b61cf939da5306d3793"
+   }
```

Comment:
Does it make sense to strip out any of this metadata stuff? No strong view, it's not as if this notebook is updated super frequently.